integration_tests/broker_test.go

```diff
@@ -114,10 +188,30 @@ var _ = Describe("Broker", func() {
 			path,
 			bytes.NewBuffer(bindingDetailsJson),
 		)
-		resp := serveRequest(responseRecorder, req)
-		Expect(responseRecorder.Code).To(Equal(http.StatusInternalServerError))
-		body := readResponseBody(resp)
-		Expect(string(body)).To(ContainSubstring("Can't bind an instance"))
+		brokerAPI.ServeHTTP(responseRecorder, req)
+		Expect(responseRecorder.Code).To(Equal(http.StatusCreated))
+
+		var data struct {
```

Comment:
We agreed to implement mongo cli connection test instead so we can assert whether values are also OK.